lib/Alchemy/Phrasea/Model/Repositories/TokenRepository.php

```diff
@@ -68,8 +68,25 @@ public function findExpiredTokens()
                 WHERE t.expiration < :date';
 
         $query = $this->_em->createQuery($dql);
-        $query->setParameters([':date' => new \DateTime()]);
+        $query->setParameters([':date' => new \DateTime('-1 month')]);
```

Comment:
no. The expiration date must be absolute, there is no reason to add a 1 month delta in a "findExpiredTokens()" method.
btw in the next method "findExpiredToken()" there is no delta.
todo : check that the expiration is SET as now()+1 month on creation.